src/main/java/lutchat/Task.java

```diff
@@ -0,0 +1,41 @@
+package lutchat;
+
+public abstract class Task {
+    boolean done;
+    String desc;
+
+    public Task(String d) {
+        desc = d;
+        done = false;
+    }
+
+    public boolean isDone() {
+        return done;
+    }
+
+    public void markAsDone() {
+        this.done = true;
+    }
+
+    public void markAsUndone() {
+        this.done = false;
+    }
+
+    public String toFileFormat() {
+        return getTaskType() + " | "
+                + (done ? "1" : "0") + " | "
+                + desc
+                + additionalDescDetailsToFileFormat();
+    }
+
+    public abstract String getTaskType();
+    public abstract String additionalDescDetailsToFileFormat();
+
+    @Override
+    public String toString() {
+        if (done) {
+            return "[X] " + desc;
+        } else
```

Comment:
`else` construct is missing brackets {}. This is not inline with the java coding standard